WordPress/Sniffs/VIP/CronIntervalSniff.php

```diff
@@ -56,7 +56,7 @@ class CronIntervalSniff extends \WordPress\Sniffs\WP\CronIntervalSniff {
 	);
 
 	/**
-	 * Don't use.
+	 * Process the token and handle the deprecation notices.
```

Comment:
I had feedback to change `Don't use` to something more descriptive, but since I'd copy pasted it I thought it would be worth going back to the source and adjusting it there, I'll undo

Reply:
The feedback applied to this PR. To the sniffs which have not been replaced by a non-deprecated version of the same.